sections/introduction.include

```diff
@@ -460,8 +460,8 @@
   </pre>
 
   HTML documents consist of a tree of elements and text. Each element is denoted in the source by
-  a <a>start tag</a>, such as "<{body|&lt;body>}>", and an <a>end tag</a>, such as
-  "<{body|&lt;/body>}>". (Certain start tags and end tags can in certain cases be <a>omitted</a>
+  a <a>start tag</a>, such as "<{body}>", and an <a>end tag</a>, such as
+  "<{body|/body}>". (Certain start tags and end tags can in certain cases be <a>omitted</a>
```

Comment:
Those are correct I just built and checked them. No need to use the &lt;. The <{}> already adds the angle brackets.